src/main/java/org/traccar/config/Keys.java

```diff
@@ -887,15 +890,27 @@ private Keys() {
             Collections.singletonList(KeyType.GLOBAL));
 
     /**
-     * Time limit for the filtering in seconds. If the time difference between last position and a new one is more than
-     * this limit, the new position will not be filtered out.
+     * If FALSE, then filter compares duplicates, distance or period only with the last received location.
+     * Server expects all locations to come sequentially.
+     *
+     * If TRUE, then filter compares duplicates, distance or period with Position's fixTime strictly before and after.
+     * Server expects locations to come at random order, since tracking device might go offline.
```

Comment:
For consistency
```suggestion
     * If false, then filter compares duplicates, distance or period only with the last received location.
     * Server expects all locations to come sequentially.
     *
     * If true, then filter compares duplicates, distance or period with Position's fixTime strictly before and after.
     * Server expects locations to come at random order, since tracking device might go offline.
```

Reply:
Can you also add a quick note that setting to true can cause potential performance issues.